libs/shared/domain/src/lib/interfaces/get-user-info-success-payload.ts

```diff
@@ -0,0 +1,3 @@
+import { UserReadModel } from '../read-models/user.read-model';
+
+export interface GetUserSuccessPayload extends UserReadModel {}
```

Comment:
Payload is only an ngrx thing move it under libs/pmp-web. Check out how the RepositoryStatistics interface works.